src/components/utrecht/OrderedListItem.test.jsx

```diff
@@ -0,0 +1,90 @@
+import { render, screen } from "@testing-library/react";
+import { createRef } from "react";
+import { OrderedList } from "./OrderedList";
+import { OrderedListItem } from "./OrderedListItem";
+import "@testing-library/jest-dom";
+
+describe("Ordered list item", () => {
+  it("renders a listitem role element", () => {
+    render(
+      <OrderedList>
+        <OrderedListItem>First item</OrderedListItem>
+      </OrderedList>
+    );
+
+    const listItem = screen.getByRole("listitem");
+
+    expect(listItem).toBeInTheDocument();
+  });
+
+  it("renders an HTML li element", () => {
+    const { container } = render(<OrderedListItem>First item</OrderedListItem>);
+
+    const listItem = container.querySelector("li:only-child");
+
+    expect(listItem).toBeInTheDocument();
+  });
+
+  it("renders a design system BEM class name", () => {
+    const { container } = render(<OrderedListItem>First item</OrderedListItem>);
+
+    const listItem = container.querySelector(":only-child");
+
+    expect(listItem).toHaveClass("utrecht-ordered-list__item");
+  });
+
+  it("displays as list-item element", () => {
+    const { container } = render(
+      <OrderedList>
+        <OrderedListItem>First item</OrderedListItem>
+        <OrderedListItem>Second item</OrderedListItem>
+        <OrderedListItem>Third item</OrderedListItem>
```

Comment:
Hmmm could we just render `OrderedListItem` instead of this big component? Or would that break the `display:list-item` test?